.tool-versions

```diff
@@ -1 +0,0 @@
-nodejs 16.14.2
```

Comment:
`.tool-versions` is for `asdf` so it should not be removed 🤔

Reply:
Reverted in [0130d9b](https://github.com/nimblehq/eslint-config-nimble/pull/46/commits/0130d9b8c3546a874151989a2073a1b529077780)